pinot-transport/src/test/java/com/linkedin/pinot/transport/common/routing/RoutingTableTest.java

```diff
@@ -268,7 +268,7 @@ public void   testCombinedKafkaRouting() throws Exception {
     ev.setState(llcSegment2.getSegmentName(), helixInstance2, consuming);
     routingTable.markDataResourceOnline(resourceName, ev, instanceConfigs);
 
-    final RoutingTableLookupRequest request = new RoutingTableLookupRequest(resourceName);
+    final RoutingTableLookupRequest request = new RoutingTableLookupRequest(resourceName, null);
```

Comment:
Please add a test case here in testCombinedKafkaRouting(), to make sure that foreHlc and foceLlc are obyed. thanks.